drivers/input/Kconfig

```diff
@@ -34,4 +34,11 @@ source "drivers/input/Kconfig.xpt2046"
 
 endmenu # Input Drivers
 
+config INPUT_MULTITOUCH_ENABLED
+    bool "Enable multitouch"
+    default n
```

Comment:
`default n` is redundant, you can drop this line

Reply:
Fixed. :)